src/h2load.cc

```diff
@@ -252,6 +255,61 @@ void rate_period_timeout_w_cb(struct ev_loop *loop, ev_timer *w, int revents) {
 } // namespace
 
 namespace {
+  // Called when the warmup duration for infinite number of requests are over
+  void warmup_timeout_cb(struct ev_loop *loop, ev_timer *w, int revents) {
+    auto client = static_cast<Client *>(w->data);
+    
+    client->warmup = 2;
+    
+    ev_timer_stop(client->worker->loop, &client->warmup_watcher);
+    std::cout << "Warm-up phase is over for client: " << client->id << std::endl;
+    
+    client->worker->stats.req_started -= client->req_started;
+    client->worker->stats.req_done -= client->req_done;
+    // we also need to adjust the failed and errored
+    client->req_todo = 0;
+    client->req_left = 1;
+    client->req_inflight = 0;
```

Comment:
It might be simpler to just avoid recording any stats before we enter the measurement phase, not sure. I'm thinking about avoiding having to reason about in flight requests, e.g when they get reset, some are still actually in flight, etc.